code/src/components/Drinks.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./drinks.css";
+
+const Drinks = ({ drink, onDrinkChange, onStepChange, onStepBackChange }) => {
+  // v1
+  return (
+    <div className="question-container">
+      <h2 className="question-title">
+        As a reality superstar you need a signature drink!
+      </h2>
+      <form className="drinks">
+        <lable className="drink-choice" for="Drinks-selection">
+          Your drink of choice
+        </lable>
+        <select
+          className="drink-selections"
+          value={drink}
+          onChange={onDrinkChange}
+        >
+          <option disabled value="">
+            --Please choose an option--
+          </option>
+          <option value="Coffe ☕️">Coffe</option>
+          <option value="Redbull ⚡️">Redbull</option>
+          <option value="Coke 🥤">Coke</option>
+          <option value="Milk 🥛">Milk</option>
```

Comment:
Maybe here it would be cool to have it the other way around so that the user can also see the emojis, and the program only needs to read letters (i.e `<option value="Redbull">Redbull ⚡</options>`